README.md

```diff
@@ -17,3 +17,14 @@ This repo includes:
 ### Nightly builds
 
 The latest built and tested packages from this repo are available on this MyGet feed: https://dotnet.myget.org/gallery/aspnetwebstack-dev
+
+### Tags and releases
+
+Git tag|Git branch|Products|MVC package|Web API package|Web Pages package
+--------|--------------|------------|------------|------------|------------
+[v2.0.4](https://github.com/aspnet/AspNetWebStack/tree/v2.0.4)|[v2.0-rtm](https://github.com/aspnet/AspNetWebStack/tree/v2.0-rtm)|ASP.NET MVC 4.0.4|4.0.40804|4.0.30506|2.0.30506
+[v2.1](https://github.com/aspnet/AspNetWebStack/tree/v2.1)||ASP.NET and Web Tools 2012.2, VS 2012 Update 2 (not on http://nuget.org)|v4 2012.2 Update RTM|v1 2012.2 Update RTM|v2 2012.2 Update RTM
+[v3.0.2](https://github.com/aspnet/AspNetWebStack/tree/v3.0.2)|[v3-rtm](https://github.com/aspnet/AspNetWebStack/tree/v3-rtm)|ASP.NET MVC 5.0.2|5.0.2|5.0.1|3.0.1
+[v3.1.3](https://github.com/aspnet/AspNetWebStack/tree/v3.1.3)|[v3.1-rtm](https://github.com/aspnet/AspNetWebStack/tree/v3.1-rtm)|ASP.NET MVC 5.1.3|5.1.3|5.1.2|3.1.2
+[v3.2.3](https://github.com/aspnet/AspNetWebStack/tree/v3.2.3)||ASP.NET MVC 5.2.3 / Web API 2.2.3 / Web Pages 3.2.3 |5.2.3|5.2.3|3.2.3
```

Comment:
Not sure why this row contained more product names than the ones above. Want something similar there? Or, perhaps do this only on the first row?

Reply:
I agree this is odd so I wouldn't have this line have a different structure than other lines. But, I also forgot that we have a mismatch for Web API between the product version and package version. Perhaps for the Web API column it should be titled `Web API package version (product version)`, and then in each row do something like `5.2.3 (2.2.3)`.